.github/workflows/python-app.yml

```
@@ -42,23 +42,8 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with nose 
       env:
-         TEST_USERNAME: fry
-         TEST_PASSWORD: fry
-         TEST_USERNAME2: zoidberg
-         TEST_PASSWORD2: zoidberg
+         USER_SETUP_JSON: tests/users.json
```

```suggestion
         USER_SETUP_JSON: hancock/tests/users.json
```

Not sure but could perhaps fix your CI failure?